codegen/templates/contrib/sets/sets.gotmpl

```diff
@@ -0,0 +1,114 @@
+package {{.Version}}
+
+import (
+    {{- if $.CustomTypesImportPath }}
+
+    . "{{ $.CustomTypesImportPath }}"
+    {{- end}}
+)
+
+// k8s resources are uniquely identified by their name and namespace
+func key(objectMeta v1.ObjectMeta) string {
+    return objectMeta.GetName() + "-" + objectMeta.GetNamespace()
```

Comment:
Any reason for using a `"-"` here. We commonly concatenate these values with a `"."` elsewhere.